### PR TITLE
[15.0][ADD] resource_boooking: Add videocall location in resource booking

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Add videocall location in resource booking as this field already exists in calendar.event
- Videocall location available to set in resource booking type. 
- Get videocall location from meeting only when available.
  
@Tecnativa

TT45337

@pedrobaeza

### CODE REVIEW

File: resource_booking/models/calendar_event.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Copyright 2021 Tecnativa - Jairo Llopis
 # Copyright 2022 Tecnativa - Pedro M. Baeza
+# Copyright 2023 Tecnativa - Carolina Fernandez
```
Comment (pedrobaeza) [inline resource_booking/models/calendar_event.py]: Don't put your copyright in a non modified file

File: resource_booking/templates/portal.xml [hunk 2]
```diff
@@ -369,6 +369,10 @@
                                 <strong>Location:</strong>
                                 <span t-field="booking_sudo.location" />
                             </div>
+                            <div class="mb-1">
+                                <strong>Meeting URL:</strong>
+                                <span t-field="booking_sudo.videocall_location" />
```
Comment (pedrobaeza) [inline resource_booking/templates/portal.xml]: This should be a link

**Comment by carolinafernandez-tecnativa** [comment]
ping @pedrobaeza

File: resource_booking/i18n/es.po [hunk 3]
```diff
@@ -33,19 +31,19 @@ msgstr ""
...
 msgid "%(partner)s - %(type)s"
-msgstr "%(partner)s - %(type)s"
+msgstr ""
 
 #. module: resource_booking
 #: code:addons/resource_booking/models/resource_booking.py:0
 #, python-format
 msgid "%(partner)s - %(type)s - %(time)s"
-msgstr "%(partner)s - %(type)s - %(time)s"
+msgstr ""
```
Comment (pedrobaeza) [inline resource_booking/i18n/es.po]: Don't empty these translations. Anyway, you can add t-translate="off" on the XML for not making this translatable, as it's not required.

**Comment by carolinafernandez-tecnativa** [comment]
@pedrobaeza is it okay now? Thanks!

**Review by pedrobaeza — APPROVED** [review]
Working as expected:

/ocabot merge minor

**Pull request merged**